examples/simple_2D.jl

```diff
@@ -1,11 +1,11 @@
 ## A simple 2D example for fluid-flow simulation
 
 using DrWatson
-@quickactivate "JutulDarcyRules-example"
+@quickactivate "JutulDarcyRules"
```

Comment:
Any reason all these example use DrWatson/quickactivate instead of only `using JutulDarcyRules` as a package?